K8s-deployment/Charts/consent-validator/example-azure-resource-values.yaml

```
@@ -0,0 +1,17 @@
+ingress:
+  hostname: cv-hostname
+  annotations: {
+    cert-manager.io/cluster-issuer: "letsencrypt-staging" # use letsencrypt-prod if proper letsencrypt certs are required
+  }
+
+consentValidator:
+  nodeSelector:
+    node.kubernetes.io/instance-type: Standard_D2as_v4
+  resources:
+    limits:
+      cpu: 500m
+      memory: 0.6Gi
```

```suggestion
      memory: 600Mi
```
Avoid writing memory in fractions, might be problematic in some places.